README.md

```diff
@@ -37,7 +37,11 @@ If you don't use `use-package`:
           "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/sourcekit-lsp")))
 ```
 
-(On macOS, you can get the path to the SourceKit-LSP executable by running `xcrun --find sourcekit-lsp` in Terminal.)
+On macOS, you can get the path to the SourceKit-LSP executable by running `xcrun --find sourcekit-lsp` in Terminal. This also works to populate the path, if your emacs shell is configured correctly: 
+
+```elisp
+(setq lsp-sourcekit-executable (string-trim-right (shell-command-to-string "xcrun --find sourcekit-lsp") "\n")))
```

Comment:
I think we can call `string-trim`, and blanks including newlines are what it trims by default, so we don't need to pass "\n". There was also a mismatched parenthesis.

```suggestion
(setq lsp-sourcekit-executable (string-trim (shell-command-to-string "xcrun --find sourcekit-lsp")))
```